spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/repository/TaskDefinitionRepository.java

```diff
@@ -36,9 +36,6 @@ public interface TaskDefinitionRepository extends KeyValueRepository<TaskDefinit
 
 	Page<TaskDefinition> findByDslTextContains(String dslText, Pageable pageable);
 
-	@Deprecated
```

Comment:
Sad that this removal did not cause any test failures 😢

Reply:
So True 😢 .  But it was dead code.